src/Uno.UI/UI/Xaml/FontFamily.cs

```diff
@@ -26,16 +23,34 @@ public FontFamily(string familyName)
 
 		public override bool Equals(object obj)
 		{
-			var fontFamily = obj as FontFamily;
-
-			if (fontFamily != null)
+			if (obj is FontFamily fontFamily)
 			{
-				return Source.Equals(fontFamily.Source);
+				return Source.Equals(fontFamily.Source, StringComparison.Ordinal);
```

Comment:
Ignore case maybe ?

Reply:
Nope.  Because it could be case sensitive on some system and lead to a different output.